Makefile

```diff
@@ -149,6 +140,24 @@ calico/felix: bin/calico-felix
 DOCKER_RUN_RM:=docker run --rm --user $(MY_UID):$(MY_GID) -v $${PWD}:/code
 DOCKER_RUN_RM_ROOT:=docker run --rm -v $${PWD}:/code
 
+DOCKER_GO_BUILD_PFX := mkdir -p .go-pkg-cache && \
+                       docker run --rm \
+                                  --net=host \
+                                  -e LOCAL_USER_ID=$(MY_UID) \
+                                  -v $${PWD}:/code \
+                                  -v $${PWD}:/go/src/github.com/projectcalico/felix:rw \
```

Comment:
> Why do we map the repo into the container under two places?

Historic artefact, I don't think it's needed any more.  I'll try to clean it up...

> make the path inside the container exactly the same as that outside.

I think that'd take quite a bit of work to make work.  (The build container already has a gopath set up that we inherit so we'd have to do more work in building the container, then do a bunch of symlinking or something).  Not sure it's worth it.